src/main/kotlin/org/jetbrains/intellij/IntelliJPlugin.kt

```diff
@@ -39,21 +39,7 @@ import org.jetbrains.intellij.dependency.PluginDependencyManager
 import org.jetbrains.intellij.dependency.PluginDependencyNotation
 import org.jetbrains.intellij.dependency.PluginProjectDependency
 import org.jetbrains.intellij.jbr.JbrResolver
-import org.jetbrains.intellij.tasks.BuildSearchableOptionsTask
-import org.jetbrains.intellij.tasks.DownloadRobotServerPluginTask
-import org.jetbrains.intellij.tasks.IntelliJInstrumentCodeTask
-import org.jetbrains.intellij.tasks.JarSearchableOptionsTask
-import org.jetbrains.intellij.tasks.ListProductsReleasesTask
-import org.jetbrains.intellij.tasks.PatchPluginXmlTask
-import org.jetbrains.intellij.tasks.PrepareSandboxTask
-import org.jetbrains.intellij.tasks.PublishPluginTask
-import org.jetbrains.intellij.tasks.RunIdeBase
-import org.jetbrains.intellij.tasks.RunIdeForUiTestTask
-import org.jetbrains.intellij.tasks.RunIdeTask
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask
-import org.jetbrains.intellij.tasks.SetupDependenciesTask
-import org.jetbrains.intellij.tasks.SignPluginTask
-import org.jetbrains.intellij.tasks.VerifyPluginTask
+import org.jetbrains.intellij.tasks.*
```

Comment:
Do we want to fold imports? I'm asking because we don't do that in other parts of the project, and if we agree on that – I'll have to fix that everywhere as well.